packages/augur-ui/src/modules/global-chat/components/global-chat.styles.less

```diff
@@ -1,111 +1,138 @@
 @import (reference) '~assets/styles/shared';
 
-.GlobalChat {
-  background: var(--color-dark-grey);
-  bottom: 0;
-  position: fixed;
-  right: 0;
-  z-index: @mask-modal;
-
-  > button {
-    background-color: var(--color-page-background);
+:root {
```

Comment:
Don't need these to be wrapped in Root, root is default -- so trading classes can remain the same except for colors `var(--` stuff. 

The betting/sports changes can remain below.